pages/common/podman-rmi.md

```diff
@@ -0,0 +1,20 @@
+# podman rmi
+
+> Remove one or more Podman images.
+> More information: <https://docs.podman.io/en/latest/markdown/podman-rmi.1.html>.
+
+- Show help:
+
+`podman rmi`
+
+- Remove one or more images given their names:
+
+`podman rmi {{image:tag}} {{image2:tag}} {{...}}`
+
+- Force remove an image:
+
+`podman rmi --force {{image}}`
+
+- Remove an image without deleting untagged parents:
+
+`podman rmi --no-prune {{image}}`
```

Comment:
```suggestion
# podman rmi

> Remove one or more Podman images.
> More information: <https://docs.podman.io/en/latest/markdown/podman-rmi.1.html>.


- Remove one or more images given their names:

`podman rmi {{{image1 image2 ...}}`

- Force remove an image:

`podman rmi --force {{image}}`

- Remove an image without deleting untagged parents:

`podman rmi --no-prune {{image}}`

- Show help:

`podman rmi`
```
I made my previous suggestion simple to commit with this comment @FearTheBadger.